src/components/DocParse.js

```diff
@@ -15,7 +17,9 @@ function parseLink (child, index) {
 	let title = child.children[0].value;
 	const linkText = child.children[0].text || linkReference || title;
 
-	// TODO: Does not work with external links!
+	if (title.indexOf('http') === 0) {
+		return <OutboundLink href={title} key={index}>{linkText}</OutboundLink>;
```

Comment:
For `href`, should we use `child.url` instead?

Reply:
Maybe?!  Seems legit.  I didn't see that field, probably because we were working with non-url entities.